cas_client/_version.py

```diff
@@ -1,2 +1,3 @@
-__version_info__ = (0, 1, 6)
+__version_info__ = (2, 0, 0)
```

Comment:
skippin' straight to 2, huh? you been studying at the [Firefox school of versioning](https://en.wikipedia.org/wiki/Firefox_version_history#Rapid_releases)? 😉

I'm curious, but it's not important.

Reply:
lol my bad, meant `1.0.0`